app/src/main/java/eu/kanade/tachiyomi/ui/extension/ExtensionPresenter.kt

```diff
@@ -67,48 +67,61 @@ open class ExtensionPresenter(
                 .sortedBy { it.pkgName }
 
         if (installedSorted.isNotEmpty() || untrustedSorted.isNotEmpty()) {
-            val header = ExtensionGroupItem(context.getString(R.string.ext_installed), installedSorted.size + untrustedSorted.size)
-            items += installedSorted.map { extension ->
-                ExtensionItem(extension, header, currentDownloads[extension.pkgName])
+            val header = ExtensionGroupItem(true, installedSorted.size + untrustedSorted.size)
+            val installedItems = mutableListOf<ExtensionItem>()
+            items += Pair(header, installedItems)
+
+            installedItems += installedSorted.map { extension ->
+                ExtensionItem(extension, header)
```

Comment:
don't set `installStep` using `currentDownloads` because

1. Item must necessarily be installed to be in this list
2. race condition between updating install step and updating list of installed items